### PR TITLE
skip synchronizing materialized view

### PR DESCRIPTION
**Pull request by emn178**
If I have a materialized view by creating manually and define the model, it will cause error when synchronizing. I want to skip it.
It's better to support define materialized view by cequel, but I think we can skip and prevent error for now.

### CODE REVIEW

File: lib/cequel/schema/table_reader.rb [hunk 1]
```diff
@@ -150,18 +173,21 @@ def read_table_settings
...
+          .keyspace(keyspace.name)
+          .table(table_name.to_s)
+      end
 
-            fail(NoSuchKeyspaceError, "No such keyspace #{keyspace.name}") if
-              !cluster.has_keyspace?(keyspace.name)
+      def cluster
+        @cluster ||= begin
+          @cluster = keyspace.cluster
+          @cluster.refresh_schema
```
Comment (emn178) [inline lib/cequel/schema/table_reader.rb]: In original table_data do this, so I also do this. Any suggestion?

Comment (pezra) [inline lib/cequel/schema/table_reader.rb]: I don't think this is a show stopper. It does only happen the first time, due to the memoization.

**Comment by emn178** [comment]
I tried to cached which keyspace refreshed, but it won't work if schema changed. This can not pass unit tests. I will try other methods later.

**Comment by pezra** [comment]
@emn178, i hope you find time to get the tests passing. This is a good change, over all, and i'd like to include it.

**Comment by pezra** [comment]
@emn178, this looks good. If you rebase it i'll merge it.

**Comment by pezra** [comment]
merge as #346. Thanks for the improvement.,

**Comment by pezra** [comment]
Released in 2.0.3. Thanks, again, for the improvement.

**Pull request closed without merging**